packages/messaging-api-messenger/src/MessengerClient.ts

```diff
@@ -426,6 +426,49 @@ export default class MessengerClient {
       .then(res => res.data, handleError);
   }
 
+  /**
+   * User Level Menu
+   *
+   * https://developers.facebook.com/docs/messenger-platform/send-messages/persistent-menu#user_level_menu
+   */
+  getUserLevelMenu(
+    userId: string,
+    { accessToken: customAccessToken }: Types.AccessTokenOptions = {}
+  ): Promise<Types.UserPersistentMenu[]> {
+    return this._axios
+      .get(
+        `/me/custom_user_settings?psid=${userId}
+        &access_token=${customAccessToken || this._accessToken}`
+      )
+      .then(res => res.data.data, handleError);
+  }
+
+  setUserLevelMenu(
```

Comment:
This should be `setCustomUserSetting(userId: string, { persistentMenu })`. The `MessengerProfile` type is not able to fit this API, for example, the following usage:

```js
setUserLevelMenu({
  getStarted: {
    payload: 'PAYLOAD',
  };
});
```